argo-cd-2.8.yaml

```diff
@@ -100,9 +105,13 @@ subpackages:
           ln -s /usr/local/bin/argocd "${{targets.subpkgdir}}"/usr/local/bin/argocd-notifications
           ln -s /usr/local/bin/argocd "${{targets.subpkgdir}}"/usr/local/bin/argocd-applicationset-controller
           ln -s /usr/local/bin/argocd "${{targets.subpkgdir}}"/usr/local/bin/argocd-k8s-auth
+    dependencies:
```

Comment:
I believe you have to rename the subbpackage names

Reply:
Done. Thanks.